backend/pcloud/pcloud.go

```diff
@@ -326,14 +330,18 @@ func NewFs(ctx context.Context, name, root string, m configmap.Mapper) (fs.Fs, e
 	f.features = (&fs.Features{
 		CaseInsensitive:         false,
 		CanHaveEmptyDirectories: true,
+		OpenWriterAt: func(ctx context.Context, remote string, size int64) (fs.WriterAtCloser, error) {
+			return newWriterAt(ctx, remote, f, size)
```

Comment:
Normally we make `OpenWriterAt` a method on `*Fs` - then it can easily read the config etc.

Reply:
done